src/components/sustainability/HeroSection.tsx

```diff
@@ -30,21 +30,83 @@ const FeatureSection = ({ textContent }: HeroSectionsProps) => {
         </div>
 
         {/* Display goals */}
-        <div className="relative grid grid-cols-2 justify-items-center pt-6">
-          {goals.map((goal, index) => {
-            const bgColor = index === 1 ? 'bg-green-25' : index === 4 ? 'bg-green-55' : 'bg-green-40';
+        <div className="grid grid-cols-3 pt-6 hidden lg:grid">
+          {goals.slice(0, 3).map((goal, index) => {
+            const bgColor = index === 1 || index === 3 ? 'bg-green-25' : 'bg-green-40';
```

Comment:
maybe this could also be more understandable by using the isEven functionality, like you did earlier in the other file

Reply:
Done